build/ci/job-template.yml

```diff
@@ -49,6 +49,7 @@ jobs:
       - script: brew update && brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/f5b1ac99a7fba27c19cee0bc4f036775c889b359/Formula/libomp.rb && brew unlink python@2 && brew install mono-libgdiplus gettext && brew link gettext --force && brew link libomp --force
         displayName: Install build dependencies
     - ${{ if and( eq(parameters.nightlyBuild, 'true'), eq(parameters.name, 'Ubuntu_x64_NetCoreApp21')) }}:
+      - script: sudo locale-gen en_US.UTF-8 && sudo update-locale
```

Comment:
This is place for nightly build only.

Use below as separate step:

- ${{ if eq(parameters.name, 'Ubuntu_x64_NetCoreApp21') }}:
      - script: sudo locale-gen en_US.UTF-8 && sudo update-locale
        displayName: Install and Update locale